packages/documentation/src/content/docs/integration/getting-started.md

```diff
@@ -112,18 +112,18 @@ The endpoint accepts a `POST` request with
 
 #### EventType
 
-| Value                              | Description                                                                 |
-| ---------------------------------- | --------------------------------------------------------------------------- |
-| `incoming_payment.created`         | Incoming payment has been created.                                          |
-| `incoming_payment.completed`       | Incoming payment is complete and doesn't accept any incoming funds anymore. |
-| `incoming_payment.expired`         | Incoming payment is expired and doesn't accept any incoming funds anymore.  |
-| `outgoing_payment.created`         | Outgoing payment was created.                                               |
-| `outgoing_payment.completed`       | Outgoing payment is complete.                                               |
-| `outgoing_payment.failed`          | Outgoing payment failed.                                                    |
-| `payment_pointer.not_found`        | A requested payment pointer was not found                                   |
-| `payment_pointer.web_monetization` | Web Monetization payments received via STREAM.                              |
-| `asset.liquidity_low`              | Asset liquidity has dropped below defined threshold.                        |
-| `peer.liquidity_low`               | Peer liquidity has dropped below defined threshold.                         |
+| Value                             | Description                                                                 |
+| --------------------------------- | --------------------------------------------------------------------------- |
+| `incoming_payment.created`        | Incoming payment has been created.                                          |
+| `incoming_payment.completed`      | Incoming payment is complete and doesn't accept any incoming funds anymore. |
+| `incoming_payment.expired`        | Incoming payment is expired and doesn't accept any incoming funds anymore.  |
+| `outgoing_payment.created`        | Outgoing payment was created.                                               |
+| `outgoing_payment.completed`      | Outgoing payment is complete.                                               |
+| `outgoing_payment.failed`         | Outgoing payment failed.                                                    |
+| `wallet_address.not_found`        | A requested wallet address was not found                                    |
+| `wallet_address.web_monetization` | Web Monetization payments received via STREAM.                              |
```

Comment:
same thing as 👇 , are we renaming the existing webhooks or just adding new ones, and deprecating the payment pointer ones?

Reply:
Same deal, https://github.com/interledger/rafiki/tree/nl-rename-all-payment-pointer should rename the existing ones. This PR will just handle docs changes. Though I see that maybe Web Monetization may not make sense for a wallet address with a non-ILP payment method.